tools/twitter-api/get-tweets.py

```diff
@@ -0,0 +1,98 @@
+from elasticsearch import Elasticsearch
+from random import randint
+import json
+import tweepy
+
+creds = json.load(open("creds.json"))
+creds = creds["twitter-creds"]
+
+consumer_key        = creds["consumer_key"]
+consumer_key_secret = creds["consumer_key_secret"]
+access_token        = creds["access_token"]
+access_token_secret = creds["access_token_secret"]
+
+auth = tweepy.OAuthHandler(consumer_key, consumer_key_secret)
+auth.set_access_token(access_token, access_token_secret)
+api = tweepy.API(auth)
+
+def GetElasticSearchHits(): 
+  es = Elasticsearch(
+    ['lp01.idea.rpi.edu:443/elasticsearch'],
```

Comment:
The address of the elasticsearch server should probably be in a config file (see the other tools config.json files). If you use a config.json you can group together all the settings needed to make the tool work, including credentials, server URIs, and any general settings.